samples/features/azure-arc-data-controller/deployment/kubeadm/ubuntu-single-node-vm/setup-controller.sh

```diff
@@ -0,0 +1,299 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run as root"
+  exit
+fi
+
+# This is a script to create single-node Kubernetes cluster and deploy Azure Arc Data Controller on it.
+#
+export AZUREARCDATACONTROLLER_DIR=aadatacontroller
+
+# Get password as input. It is used as default for controller, SQL Server Master instance (sa account).
+#
+while true; do
+    read -s -p "Create Password for Azure Arc Data Controller: " password
+    echo
+    read -s -p "Confirm your Password: " password2
+    echo
+    [ "$password" = "$password2" ] && break
+    echo "Password mismatch. Please try again."
+done
+
+# Name of virtualenv variable used.
+#
+export LOG_FILE="aadatacontroller.log"
+export DEBIAN_FRONTEND=noninteractive
+
+# Requirements file.
+export AZDATA_PRIVATE_PREVIEW_DEB_PACKAGE="https://aka.ms/aadatacontrollerazdata"
+
+# Kube version.
+#
+KUBE_DPKG_VERSION=1.15.0-00
```

Comment:
You could change to 1.16.3 - this is latest. 1.15 is from June 2019.

Reply:
Done.